### ISSUE TITLE
How to exclude specific files from the sitemap (sorry for insisting)

### ISSUE THREAD

**Issue by ornelasnarciso** [Quasibit/eleventy-plugin-sitemap#36]
How do i exclude specific files from the sitemap, that don't have front matter, like "footer"? I'm a self taught programmer and I have no idea what to do with the information that's on the docs. 

eleventyConfig.addCollection("sitemap", function(collectionApi) {
  return collectionApi.getAll().filter(item => {

    // Place your condition here for excluding items from the sitemap. --> What condition? What do i put here if i want to remove the footer.njk and other files? 

    return true;
  });
});

Thank you for your time.

**Comment by nunof07** [comment]
Your question falls outside the scope of this plugin.

There are several ways to do what you want. I can't possibly know the exact details with the information you provided.

If you don't know what to do with the information I provided in the previous issue, you probably need to learn more about programming or Eleventy specifically.

Please don't open new issues for the same thing. You can comment on closed issues as well.

**Comment by ornelasnarciso** [comment]
It's a specific question about the plugin. I wanted to remove a page in which I'm unable or it's not possible to create front matter, like /partials/footer.njk. That is not specified on the docs.  

Since I was able to connect everything else, i'll try to figure it out how to do this. I'll eventually make it. :) 

Great plugin and thank you for your time.

**Comment by nunof07** [comment]
You don't need to use front matter to add meta data to a file. You can add [Template & Directoy Data](https://www.11ty.dev/docs/data-template-dir/), which is also mentioned in the docs. That's how Eleventy works - data cascades from different levels.

So a better question is "How do I add data to a file without front matter?" - then you realize it is a generic question about Eleventy, not specific to this plugin.

We can't possibly include all possible variations and use cases in the docs of this plugin, but I'd be happy to check a PR if you think the docs can be improved in this regard.

And thanks for the kind words. 👍

**Comment by ornelasnarciso** [comment]
I'll try to explain. :D 

I have a layout for posts. And i like to create files like "author.njk" and call them to the posts.njk layout like this {% include "partials/author.njk" %}. It's easier for me to read the code and update something if I have to. The problem is that i can't create front matter for the author.njk file and remove it from the sitemaps. So something like this will appear on sitemaps: https://ornelasnarciso.com/partials/author/ 

The workaround I found was to remove all these links manually on google search console, so somehow I solved the problem. :D 

Your plugin save me a lot of time. Thanks.
